gradle.properties

```diff
@@ -5,4 +5,9 @@ api_gen_version=0.15.62
 model_api_version=1.0.+
 json_bulk_access_version=2.3
 ktor_version=2.0.3
-micronautVersion=3.5.3
\ No newline at end of file
+micronautVersion=3.5.3
+quarkusPluginId=io.quarkus
+quarkusPluginVersion=2.14.0.Final
+quarkusPlatformGroupId=io.quarkus.platform
+quarkusPlatformArtifactId=quarkus-bom
+quarkusPlatformVersion=2.14.0.Final
```

Comment:
The underscore variables were generated automatically. The new properties use camel case ... So either we do underscores or camel case, lets unify it either way. I have no preference.

`mpsExtensions_version` is the worst offender here :laughing:

Reply:
The major part of this project is based on the Quarkus Gradle project generator. Is it a good idea to diverge from the default way such a Quarkus project is generated? Btw, even IntelliJ complains about the non-canonical underscore variable names used in Gradle.

Reply:
I just adapted this. The project moves farther away from the generated on in any case.